python/lsst/ts/vent/controller/controller.py

```diff
@@ -22,33 +22,49 @@
 import logging
 from enum import IntEnum
 
-import megaind
+try:
+    import megaind
+
+    MEGAIND_AVAILABLE = True
+except ImportError:
+    MEGAIND_AVAILABLE = False
 import pymodbus.client
 
 from . import vf_drive
 from .config import Config
+from .simulate import DomeVentsSimulator
 
-__all__ = ["Controller"]
+__all__ = ["Controller", "VentGateState"]
 
 
 class VentGateState(IntEnum):
-    CLOSED = 0
-    PARTIALLY_OPEN = 1
-    OPEN = 2
+    CLOSED = 1
+    PARTIALLY_OPEN = 2
+    OPEN = 3
     FAULT = -1
 
 
+class FanDriveState(IntEnum):
+    STOPPED = 1
+    OPERATING = 2
+    FAULT = 3
+
+
 class Controller:
     """A controller that commands the components associated with the AT dome
     vents and fans. The code in this class is meant to run on the Raspberry Pi
     described in
     https://confluence.lsstcorp.org/display/~fritzm/Auxtel+Vent+Gate+Automation.
     """
 
-    def __init__(self, config: Config = Config()):
+    def __init__(self, config: Config = Config(), simulate: bool = False):
         self.cfg = config
         self.default_fan_frequency = self.cfg.max_freq
         self.log = logging.getLogger(type(self).__name__)
+        self.simulate = simulate
```

Comment:
Do you really need to store the `simulate` flag? I think you can determine if it is in simulation mode by checking it `self.simulator is None`.